### PR TITLE
[test] : add unittests for vpc.go file

### PR DESCRIPTION
**Pull request by rahulait**
### General:

* [x] Have you removed all sensitive information, including but not limited to access keys and passwords?
* [x] Have you checked to ensure there aren't other open or closed [Pull Requests](../../pulls) for the same bug/feature/question?

### Pull Request Guidelines:

1. [ ] Does your submission pass tests?
1. [ ] Have you added tests? 
1. [x] Are you addressing a single feature in this PR? 
1. [x] Are your commits atomic, addressing one change per commit?
1. [x] Are you following the conventions of the language? 
1. [x] Have you saved your large formatting changes for a different PR, so we can focus on your work?
1. [ ] Have you explained your rationale for why this feature is needed? 
1. [ ] Have you linked your PR to an [open issue](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)

### CODE REVIEW

File: cloud/linode/vpc_test.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+			got := GetAllVPCIDs()
+			sort.Ints(got)
+			if !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("GetAllVPCIDs() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestGetVPCID(t *testing.T) {
```
Comment (komer3) [inline cloud/linode/vpc_test.go:46]: qq: any reason to not use tabular test format for this?

Comment (rahulait) [reply cloud/linode/vpc_test.go:46]: `ListVPCs()` call and its response is changing in each test run, hence I avoided tabular format. Not sure if we can achieve that with tabular format unless we do if/else with bunch of conditions to decide which mock call to use.

Comment (komer3) [reply cloud/linode/vpc_test.go:46]: Ah i see. I'm okay with that

**Review by komer3 — APPROVED** [review]
LGTM!

**Pull request merged**